app/views/users/index.html.erb

```diff
@@ -1,19 +1,20 @@
-
-    <div class="w-75 mx-auto mt-5">
-      <div class="card">
-      <div class="card-header">
-      <h4 class="fw-dark">Users:<strong><%= @users.count%></strong></h4>
-      </div>
-          <% @users.each do |u|%>
-     <div class="card-body">
-       <div class="d-flex justify-content-between">
-         <div>
-             <h3>Username: <%= u.name %></h3>
-             <%= link_to 'show user', user_path(u), class: 'btn-sm btn btn-primary' %>
-         </div>
-         <p class=""> Number of posts: <%= u.posts.count %> </p>
+<div class="container">
+  <div class="card" style="width: 100%">
+    <ul class="list-group list-group-flush">
+      <% @users.each do |user|%>
+      <li class="list-group-item ">
+        <div class="d-flex justify-content-between">
+          <div class="h5"><b>Name: </b><%=user.name%></div>
+        <small>Posts: </b><%=user.posts_counter || 0 %></b> </small>
         </div>
-      </div>
-      <% end %>
-      </div>
+        <div class="">
+          <%= link_to 'show user', user_path(user), class: 'btn-sm btn btn-primary' %>
+         </div>
+      </li>
+      <%end%>
```

Comment:
- Please note that when I populated the database to see the form created, I was met with an empty page. This might be an issue with routing. Kindly look into this 👍🏽
![Screenshot from 2023-03-03 16-46-58](https://user-images.githubusercontent.com/44624138/222764628-9af7f97a-d1ee-4fd4-80ae-2baed2dc1a39.png)

Reply:
- I get an error when I try adding a user from the app page. This might be a problem with your routes, kindly look into it 😉
![Screenshot from 2023-03-03 16-51-41](https://user-images.githubusercontent.com/44624138/222765538-1bd646ac-0c1a-4cda-a209-88b2d40daa37.png)

Reply:
sorry, i just added the link but, adding users  is not required in the project requirement

Reply:
The page show navbar and footer because there is no user yet, when a user is added either by rails command you can be able to navigate to adding post, comments and likes